src/components/Search.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {GestureResponderEvent, StyleProp, View, ViewStyle} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@styles/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import {PressableWithFeedback} from './Pressable';
+import Text from './Text';
+import Tooltip from './Tooltip';
+
+type SearchProps = {
+    // Callback fired when component is pressed
+    onPress: (event?: GestureResponderEvent | KeyboardEvent) => void;
+
+    // Text explaining what the user can search for
+    prompt: string;
+
+    // Text showing up in a tooltip when component is hovered
+    tooltip?: string;
+
+    // Styles to apply on the outer element
+    style?: StyleProp<ViewStyle>;
+};
+
+function Search({onPress, prompt, tooltip, style}: SearchProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    return (
+        <Tooltip text={tooltip ?? translate('common.search')}>
+            <PressableWithFeedback
```

Comment:
Hey @aimane-chnaif . If you change the sidebar background color from `sidebar: colors.productDark200,` to `sidebar: colors.productDark100,` you will see the touchable feedback in action. It's visible on `onPress`. There's no style change when `onPress` becomes `onLongPress`.

@dannymcclain @shawnborton LMK if this looks good.

https://github.com/Expensify/App/assets/85628901/a57f0760-41df-4119-aa98-4375136d1e04

Reply:
Ah yeah a couple of things here:
- At some point we need to change this entire sidebar to use the same color100 (appBG) color. Not sure when or where we plan to do that.
- The search input should technically be using color300 (same as rowHover) as the background color. I don't think we've decided anything for the search inputs :hover or :active but I suppose it could be color400 for hover and color500 for active/press?

Reply:
Asked Maciej to address this in the follow up PRs

Reply:
Oh, the color of the search input from figma (#07271F) seems to be gone from the repo after recent changes in the theme. So I'll do 300/400/500 @shawnborton. Although our app now suggests something like that:
```
    componentBG: colors.productDark100,
    hoverComponentBG: colors.productDark200,
    activeComponentBG: colors.productDark400,
```
But I know it can change as currently we have the wrong background color in the sidebar, while ideal-nav will have bg200 if I remember correctly

Reply:
While on the buttons currently we have:
```
    buttonDefaultBG: colors.productDark400,
    buttonHoveredBG: colors.productDark500,
    buttonPressedBG: colors.productDark600,
```

Reply:
@shawnborton 

https://github.com/Expensify/App/assets/85628901/8834d328-002f-447b-ab2f-fdbbe7c8578d


https://github.com/Expensify/App/assets/85628901/fac5b408-eff6-41ba-94e2-e2d0ce333c5f

Reply:
@shawnborton 

https://github.com/Expensify/App/assets/85628901/2185c97f-07ec-4c66-97cc-4cfbbdc8abdd

Reply:
@shawnborton 


https://github.com/Expensify/App/assets/85628901/9305c399-1443-4f3a-bf23-dd4607b438b3